### ISSUE TITLE
Add the ability to customize a post's hero

### ISSUE THREAD

**Issue by andy-wojciechowski** [msoe-sse/jekyll-post-editor#42]
Currently posts created with the post editor are limited to the following background image: https://source.unsplash.com/collection/145103/. We should add the ability to change the background image since not all posts currently on the SSE website use that background image. The hero would not be a required field to submit a post, just an option for a user to add customization to their post and we could fall back to the hero that we're using currently if they don't supply a hero.

There are a couple of ways we could implement this customization. My thought is we would break it into 2 phases.

Phase 1: Add the ability for a custom hero from a URL. This would be the most easy customization we could add. This would involve a textbox for a user to paste an image URL into and then we would substitute that URL when we format the post.

Phase 2: Add the ability for a custom hero based on an image file. This would be the more complicated phase. We could build on our existing image filesystem using carrierwave and add drag and drop support to this textbox or add an upload button. That custom image file would then need to be committed to the website's asset folder then.

**Comment by andy-wojciechowski** [comment]
@GFLEMING133 I assigned myself to this issue to handle Phase 2, but somebody else could also easily be assigned to this issue to implement phase 1 of this issue.

**Comment by GFLEMING133** [comment]
Yes, I think phase 1 would be an excellent choice for a newer developer. It's a relatively simple feature and it should be easily testable.

**Comment by andy-wojciechowski** [comment]
That's a good point, I'll still leave my name on this issue to show how CarrierWave works to a newer developer then.

**Comment by andy-wojciechowski** [comment]
This first phase of this issues is done. It was released on 10/3
